LEGO1/mxdisplaysurface.cpp

```diff
@@ -174,7 +174,7 @@ void MxDisplaySurface::SetPalette(MxPalette *p_palette)
 }
 
 // OFFSET: LEGO1 0x100bc200 STUB
-void MxDisplaySurface::vtable24(undefined4, undefined4, undefined4, undefined4, undefined4, undefined4, undefined4, undefined4)
+void MxDisplaySurface::vtable24(DDSURFACEDESC&, undefined4, undefined4, undefined4, undefined4, undefined4, undefined4, undefined4)
```

Comment:
This is something I missed previously, but could you change the interface of this and all the other functions (I believe it's `vtable2c` and the two Setup functions in the transition manager) to take the COM pointer version, `LPDDSURFACEDESC` instead of a reference to `DDSURFACEDESC`? This doesn't change anything functionally but would be more consistent with how they have used the DirectX APIs in other places. Thanks!